<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coquillo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../coquillo.qrc">
    <normaloff>:/resources/Coquillo.png</normaloff>:/resources/Coquillo.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>10</horstretch>
     <verstretch>10</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTableView" name="items">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCoquillo">
    <property name="title">
     <string>Coquillo</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionRecursiveScan"/>
    <addaction name="separator"/>
    <addaction name="actionToggleMenuBar"/>
    <addaction name="actionToggleStatusBar"/>
    <addaction name="actionToggleMainToolBar"/>
    <addaction name="separator"/>
    <addaction name="actionToggleLocationDock"/>
    <addaction name="actionToggleEditorDock"/>
    <addaction name="separator"/>
    <addaction name="actionLock_interface"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionManage_bookmarks"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuCoquillo"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockEditor">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tag Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="EditorWidget" name="editor">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLocation">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Location</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="DirectorySelectorWidget" name="location">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionLevel_up"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionReset"/>
   <addaction name="actionReloadOrAbort"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_all"/>
   <addaction name="actionSelect_inverse"/>
   <addaction name="separator"/>
   <addaction name="actionToggleProcessor"/>
   <addaction name="actionToggleCddb"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Coquillo...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Select previous track</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Select next track</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset resource="../../coquillo.qrc">
     <normaloff>:/resources/icons/select_all_48x48.png</normaloff>:/resources/icons/select_all_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all tracks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelect_inverse">
   <property name="icon">
    <iconset resource="../../coquillo.qrc">
     <normaloff>:/resources/icons/select_inverse_48x48.png</normaloff>:/resources/icons/select_inverse_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Inverse</string>
   </property>
   <property name="toolTip">
    <string>Invert selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save all changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Revert all changes</string>
   </property>
  </action>
  <action name="actionReloadOrAbort">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionToggleProcessor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Processor</string>
   </property>
   <property name="toolTip">
    <string>Go to the parent directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionRecursiveScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scan subdirectories</string>
   </property>
  </action>
  <action name="actionToggleMenuBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Menubar</string>
   </property>
   <property name="toolTip">
    <string>Toggle menubar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionToggleCddb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CDDB</string>
   </property>
   <property name="toolTip">
    <string>Open CDDB search dialog</string>
   </property>
  </action>
  <action name="actionNoBookmarks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No bookmarks</string>
   </property>
  </action>
  <action name="actionBookmark">
   <property name="text">
    <string>Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Add or remove this directory in bookmarks</string>
   </property>
  </action>
  <action name="actionLock_interface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock interface</string>
   </property>
   <property name="toolTip">
    <string>Lock toolbars and docks</string>
   </property>
  </action>
  <action name="actionToggleStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statusbar</string>
   </property>
  </action>
  <action name="actionLevel_up">
   <property name="text">
    <string>Up</string>
   </property>
   <property name="toolTip">
    <string>Go to the parent directory</string>
   </property>
  </action>
  <action name="actionManage_bookmarks">
   <property name="text">
    <string>Manage...</string>
   </property>
  </action>
  <action name="actionToggleEditorDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tag Editor</string>
   </property>
  </action>
  <action name="actionToggleLocationDock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </action>
  <action name="actionToggleMainToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectorySelectorWidget</class>
   <extends>QWidget</extends>
   <header>DirectorySelectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditorWidget</class>
   <extends>QWidget</extends>
   <header>EditorWidget.h</header>
   <container>1</container>
   <slots>
    <slot>setSelection(QItemSelection)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../coquillo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleMenuBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>menubar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_all</sender>
   <signal>triggered()</signal>
   <receiver>items</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLock_interface</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setInterfaceLocked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleStatusBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>statusbar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManage_bookmarks</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showBookmarkDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleEditorDock</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockEditor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleLocationDock</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockLocation</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleMainToolBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectNextIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevious</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectPreviousIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_inverse</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectInverse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConfigure</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSettingsDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setInterfaceLocked(bool)</slot>
  <slot>showMainMenu()</slot>
  <slot>showBookmarkDialog()</slot>
  <slot>showAboutDialog()</slot>
  <slot>selectNextIndex()</slot>
  <slot>selectPreviousIndex()</slot>
  <slot>selectInverse()</slot>
  <slot>showSettingsDialog()</slot>
 </slots>
</ui>
